mycore-ocfl/src/main/java/org/mycore/ocfl/MCROCFLPersistenceTransaction.java

```diff
@@ -46,9 +46,9 @@ public class MCROCFLPersistenceTransaction implements MCRPersistenceTransaction
 
     private static final Logger LOGGER = LogManager.getLogger();
 
-    private static final MCROCFLXMLClassificationManager MANAGER = MCRConfiguration2
-        .<MCROCFLXMLClassificationManager>getSingleInstanceOf("MCR.Classification.Manager")
-        .orElse(null);
+    private static final MCROCFLXMLClassificationManager MANAGER
+        = MCRConfiguration2.<MCROCFLXMLClassificationManager>getSingleInstanceOf("MCR.OCFL.Classification.Manager")
+            .orElseThrow();
```

Comment:
I thought we agreed to make it static to the ocfl classification manager, since this ocfl specific implementation is silly to make it exchangeable by anything else. This just adds the additional factor that something can go wrong if this setting is set wrong. On top of that, the different implementations are, or should be, toggled by the EventHandler, not by the Manager (except Metadata due to its different implementation).